noggin/utility/__init__.py

```diff
@@ -60,10 +60,11 @@ def fn(*args, **kwargs):
 
 
 def group_or_404(ipa, groupname):
-    try:
-        return ipa.group_show(groupname)
-    except python_freeipa.exceptions.NotFound:
+    group = ipa.group_find(groupname)['result']
+    if ipa.group_find(groupname)['result'] == []:
```

Comment:
You're doing the call twice, why not just test `group` ? Also, you can test that a list is empty by just doing `if not groups:`

Reply:
Ah, I added the var for that reason and just forgot to use it :man_facepalming: